src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,70 @@
+.header{
+    width: 100%;
+    height: 72px;
+    box-sizing: border-box;
+    position: absolute;
+    left: 0%;
+    right: 0%;
+    top: 0%;
+    bottom: 0%;
+    padding-right: 150px;
+    padding-left: 150px;
+    background: #FFFFFF;
+    border: 1px solid #DDE2FF;
+    border-radius: 5px;
+}
+
+.header-content{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 100%;
+}
+.header-logo-label{
+    display: flex;
+    vertical-align: middle;
+}
+.header-label{
+    font-size: 36px;
+    color: #7A8DFD;
+    padding-left: 14px;
+}
+.header-search button{
+    background-color: white;
+    border: none;
+}
+.header-search img{
+    height: 24px;
+}
+
+// Small devices 
+@media (max-width: 768px) { 
+    .header{
+        height: 56px;
+        padding-right: 20px;
+        padding-left: 20px;
+    }
+    .header-label{
+        font-size: 26px;
+        color: #7A8DFD;
+        padding-left: 8px;
+    }
+ }
+
```

Comment:
can we use [@mixin](https://sass-lang.com/documentation/at-rules/mixin) for code reuse purpose

Reply:
will exacute mixin in the next resolve commit apu